tailwind.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  future: {
+    // removeDeprecatedGapUtilities: true,
+    // purgeLayersByDefault: true,
+  },
+  purge: {
+    enable: true, // To test locally.
+    layers: ['components', 'utilities'],
+    content: ['./templates/**/*.html', './templates/**/*.ejs'],
+    options: {
+      keyframes: true,
+    },
+  },
+  theme: {
+    extend: {
+      outline: {
+        ocha: ['2px solid var(--cd-bright-blue)', '1px'],
+      },
+    },
+  },
```

Comment:
per our conversation, `theme` can be cut as our use-case doesn't seem to require this complexity